internal/querycoordv2/job/job_load.go

```diff
@@ -205,6 +196,15 @@ func (job *LoadCollectionJob) Execute() error {
 		return errors.Wrap(err, msg)
 	}
 
+	// 5. update next target, no need to rollback if pull target failed, target observer will pull target in periodically
+	_, err = job.targetObserver.UpdateNextTarget(req.GetCollectionID())
+	if err != nil {
+		msg := "failed to update next target"
+		log.Error(msg, zap.Error(err))
+		return errors.Wrap(err, msg)
+	}
+	job.undo.TargetUpdated = true
```

Comment:
This will not work well if error occurs:
returning error will trigger a series of rollback operations, and response with error to the client.
The rollback doesn't contains remove the collection/partition from meta, so the system will be confused